spec/v1/annotations/object-model.yaml

```
@@ -31,6 +31,18 @@ definitions:
         title: Element Reference
         ref: "#/definitions/ElementReference"
 
+  "@ObjectModel.custom":
+    type: boolean
+    description: |-
+      Added through custom development, diverges from the "core" model.
```

I am wondering whether we can be more precise here. For instance, 

- What does "diverges from the core model" mean?
- Is `@ObjectModel.customer: false` the same as annotation not present?
- Is it "It is a custom field if annotation is set to true." or "It is a custom field iff annotation is set to true."

```suggestion
If set to true, it is a custom element (field, entity, service, etc.). If set to false, it is not a custom element. If annotation not present, then it is not defined whether it is a custom element.
```

---

Yes, "diverges from the core model" is vague. I like "custom element" but it's also not more explanatory. 

I agree, if annotation is not present, we "don't know".